app/_redirects

```diff
@@ -3,6 +3,14 @@
 # More general/sweeping redirects should be at the bottom.
 
 
+
+
+# Konnect rename
+
+/konnect/runtime-manager/* /konnect/gateway-manager/:splat
+/konnect/runtime-manager/runtime-instances/* /konnect/gateway-manager/data-plane-nodes/:splat
```

Comment:
Not sure if this one is working correctly. I tried `/konnect/runtime-manager/runtime-instances/renew-certificates/` and `/konnect/runtime-manager/runtime-instances/upgrade/` and they both resulted in 404 instead of redirecting to `data-plane-nodes`. The `gateway-manager` part redirected correctly though, just the `data-plane-nodes` part didn't.

Reply:
^ These two redirects are in the wrong order. A more specific redirect should come before a general redirect. So 
`/konnect/runtime-manager/*` should be the last entry in this list.